java-rmi/test/com/baeldung/rmi/JavaRMIIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.rmi;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+import org.junit.Test;
+
+public class JavaRMIIntegrationTest {
+	
+	@Test
+	public void WhenRunServer_thenServerStarts() {
```

Comment:
Does it concern you that JUnit doesn't guarantee the order in which these methods will execute? Maybe converting this into a @Before or using the @FixMethodOrder annotation and then changing the name of the server one so that they can run in lexical order.